src/sagemaker/processing.py

```diff
@@ -442,6 +442,34 @@ def __init__(
             network_config=network_config,
         )
 
+    def get_run_args(
```

Comment:
Does this method need to be an abstract method?

Reply:
I tried but don't see a way of making this work.

The class inheritance looks like this: `Processor>ScriptProcessor>_SparkProcessorBase>PySparkProcessor`

ScriptProcessor can't be an abstract class because it's instantiated and used on its own in some cases. I can make the Processor class abstract and give it an abstract `get_run_args()`, but that only requires `ScriptProcessor` to implement it, I see no way of forcing subclasses (ie, _SparkProcessorBase, PySparkProcessor) to implement an abstract method for the super of its super. Correct me if I'm wrong. 

Moreover, there are cases like SKLearn where it doesn't need to be implemented. by the subclass. Its inheritance is `Processor>ScriptProcessor>SKLearnProcessor`, so they `get_run_args()` on `ScriptProcessor` works fine for it.